sdk/include/opentelemetry/sdk/metrics/ungrouped_processor.h

```diff
@@ -0,0 +1,246 @@
+#pragma once
+
+#include <map>
+#include "opentelemetry/sdk/metrics/aggregator/counter_aggregator.h"
+#include "opentelemetry/sdk/metrics/aggregator/exact_aggregator.h"
+#include "opentelemetry/sdk/metrics/aggregator/gauge_aggregator.h"
+#include "opentelemetry/sdk/metrics/aggregator/histogram_aggregator.h"
+#include "opentelemetry/sdk/metrics/aggregator/min_max_sum_count_aggregator.h"
+#include "opentelemetry/sdk/metrics/aggregator/sketch_aggregator.h"
+#include "opentelemetry/sdk/metrics/processor.h"
+#include "opentelemetry/sdk/metrics/record.h"
+#include "opentelemetry/version.h"
+
+#define stringer(x) (#x)
```

Comment:
It'll cause macro pollution.

Reply:
I've never used macros before, how should I change this?

Reply:
Do I just need to #undef string at EOF?

Reply:
In addition, avoid a common name as it might cause conflict with other headers.

e.g. `#define UNGROUPED_PROCESSOR_STRINGER`.

Stepping back, need to evaluate the pros and cons and if it doesn't pay much, don't use it 😄

Reply:
Moved it over to ungrouped_processor.cc, let me know if this is better. I think it pays a little bit as we don't have to create a map.